waspc/data/Generator/templates/react-app/src/auth/pages/createAuthRequiredPage.jsx

```diff
@@ -4,25 +4,31 @@ import React from 'react'
 import { Redirect } from 'react-router-dom'
 import { useAuth } from 'wasp/client/auth'
 
+{=# pageLoader.isDefined =}
+{=& pageLoader.importStatement =}
+// TODO: Do we have a way here to simply provide an alias?
```

Comment:
That's why I should finish this thing, it's close to being completed! https://github.com/wasp-lang/wasp/pull/1865